### DISCUSSION TITLE
Unminified Javascript Distribution Files

### DISCUSSION THREAD

**Discussion by blackdrago** [tighten/ziggy#539] [General]
#### Description

Would it be possible to get unminified versions of the Javascript distribution file(s)? It would make debugging 10x easier on local/dev instances.

#### Suggestion

Simply having unminified files in the `/dist/` directory would make it possible.

**Comment by bakerkretzmar** [comment]
You can generate these locally for debugging by running `npm install` and `npm run build -- --no-compress` from the root of this package (e.g. `vendor/tightenco/ziggy`).

**Comment by blackdrago** [comment]
Thank you -- this does help me, but it adds a lot of additional steps to my asset management system (namely having to run `npm install` and `npm run build -- --no-compress` against a vendor directory) to generate an unminified file, just so my asset management system can be in charge of when it is or is not minified. If the unminified files were included in `dist` (or another directoy), then I could simply refer to those without additional install/build steps.

**Reply by bakerkretzmar** [reply]
What kind of issues are you debugging that you need to look in Ziggy's source for? Are there errors you're seeing that you think are bugs that we should work on fixing?
